app/controllers/projects/members_controller.rb

```diff
@@ -2,9 +2,8 @@
 
 module Projects
   # Controller actions for Members
-  class MembersController < ApplicationController
+  class MembersController < Projects::ApplicationController
     include MembershipActions
-
     layout 'projects'
```

Comment:
Since you are basing this off of Projects::ApplicationController you can remove the layout line.